### ISSUE TITLE
Add languages in parsers

### ISSUE THREAD

**Issue by LePetitTim** [GeotrekCE/Geotrek-admin#1815]

**Comment by camillemonchicourt** [comment]
Yes, would be useful for APIDAE parser as an example as APIDAE data are available in several language but we import only one.

**Comment by LePetitTim** [comment]
Exactly.

**Comment by camillemonchicourt** [comment]
New PR in progress : https://github.com/GeotrekCE/Geotrek-admin/pull/2308

**Comment by camillemonchicourt** [comment]
First part included in version 2.89.0 : Partially handle translated fields: when setting ``fill_empty_translated_fields`` to ``True``, all empty translation fields for all languages will be set with the parsed value
